gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
```

Comment:
Hey, guys. Yes! There is. We currently use a config.js file in the UXPL (https://github.com/edx/ux-pattern-library/blob/master/gulp/config.js) and its how I've set up my personal FED starter (https://github.com/talbs/mise-en-place/blob/master/gulp/config.js).

With each task-based file, we then only set the config and package vars we need and can also call other tasks to run as dependencies (e.g. minifying css https://github.com/talbs/mise-en-place/blob/master/gulp/tasks/css-minify.js).

- - -

On a related note, I ripped a lot of this off of https://github.com/vigetlabs/gulp-starter, which has since been really improved to be a really robust and pluggable static asset pipeline. I'm trying to parse what Viget's doing there now, but could use some help with the heavier levels of JS/abstraction logic. Looks interesting if anyone wants to dig in.